.gitattributes

```diff
@@ -0,0 +1,4 @@
+package-lock.json linguist-generated=true -diff
+yarn.lock linguist-generated=true -diff
```

Comment:
Projects should probably only have one or the other of `package-lock.json` & `yarn.lock`. Whichever one this project doesn't use can be remove here.

Reply:
Yeah this was just a copy paste from naj